### ISSUE TITLE
Not able to build Rive 0.2.4 in Unreal 5.5

### ISSUE THREAD

**Issue by bubble8773** [rive-app/rive-unreal#131]
Hello Tod, 

I am trying to build Rive 0.24 on Unreal 5.5 on visual studio but I am getting the following errors.

`The command ""D:\Epic Games\UE_5.5\Engine\Build\BatchFiles\Build.bat" RIVECppProjectEditor Win64 Development -Project="D:\Unreal Projects2\RiveTest\RIVECppProject\RIVECppProject.uproject" -WaitMutex -FromMsBuild -architecture=x64" exited with code 6.`
`Cannot open include file: 'RHIResourceUpdates.h': No such file or directory ( from RenderContextRHIImpl.cpp Line 9)`
`Cannot open include file: 'WindowsTargetSettings.h': No such file or directory ( form RiveEditorModule.cpp Line 23)"
` 

I would greatly appreciate your help. 
Thanks

**Comment by sshumaker** [comment]
Hey @schetle - any ETA on this fix? Obviously a showstopper for everyone.

**Comment by Tod-Rive** [comment]
Hi all, taking a look at this now.

**Comment by sshumaker** [comment]
Hey @Tod-Rive - any updates here or ETA?

**Comment by Tod-Rive** [comment]
Yes! Just waiting on some last changes before we can do a release. No ETA unfortunately, but we're close.

**Comment by Spexso** [comment]
Hey @Tod-Rive , Any updates on 5.5 support ?

**Comment by Tod-Rive** [comment]
Still working on it. There have been some fairly significant changes in the runtime so it's taking a bit longer than normal. Once I get a clean build I can release the package.

**Comment by Tod-Rive** [comment]
0.2.6 has been released with 5.5 support, closing this issue.
